spec/requests/api/reservations_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/reservations', type: :request do
+  path '/api/users/{user_id}/reservations' do
+    # You'll want to customize the parameter types...
+    parameter name: 'user_id', in: :path, type: :string, description: 'user_id'
+
+    get('list reservations') do
+      response(200, 'successful') do
+        let(:user_id) { '123' }
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+
+    post('create reservation') do
+      response(200, 'successful') do
+        let(:user_id) { '123' }
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+        run_test!
+      end
+    end
+  end
+
+  path '/api/users/{user_id}/reservations/{id}' do
```

Comment:
Please change the path to `'api/reservations/{id}'`.
The path does not require `motor_id` or `user_id`.

Reply:
Thank you for the feedback; I will implement the required changes for the motorcycle and reservation integration test.